Mathlib/Algebra/Order/Ring/Canonical.lean

```diff
@@ -53,6 +53,14 @@ instance (priority := 100) toMulLeftMono [NonUnitalNonAssocSemiring R]
   rw [mul_add]
   apply self_le_add_right
 
+-- see Note [lower instance priority]
+instance (priority := 100) toMulRightMono [NonUnitalNonAssocSemiring R]
+    [LE R] [CanonicallyOrderedAdd R] : MulRightMono R := by
+  refine ⟨fun a b c h => ?_⟩; dsimp [swap]
```

Comment:
```suggestion
  refine ⟨fun a b c h => ?_⟩
  dsimp [swap]
```
I don't see why this should be on a single line

Reply:
I just copied from the proof above and I don't see why as well. I changed both of them.